.github/workflows/dev.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build_backend_cerestats_api:
-    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-push-docker.yaml@master
+    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-push-docker.yaml@1.0.0
```

Comment:
Why do I see these changes here? Fix it.

Reply:
fixed